config.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+// configuration module for cert-checker
+//
+// Copyright (c) 2022 - Valentin Kuznetsov <vkuznet AT gmail dot com>
+//
+import (
+	"encoding/json"
+	"log"
+	"os"
+)
+
+type Configuration struct {
+	Keytab string `json:"keytab"` // keytab file
```

Comment:
please add validation and omit parameters, e.g.
```
type Configuration struct {
	Keytab string `json:"keytab",omitempty` // keytab file
....
}
```
This will allow to skip some in configuration json